features/auth/useAuthStore.ts

```diff
@@ -51,8 +59,10 @@ export default function useAuthStore() {
   );
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState<string | null>(null);
-  const [flowState, setFlowState] =
-    usePersistedState<SignupFlowRes.AsObject | null>("auth.flowState", null);
+  const [flowState, setFlowState] = usePersistedState<SignupFlow | null>(
```

Comment:
@darrenvong this is where the flow state is read from localstorage. If I'm not wrong the `useEffect` you were referring to was only for the activation link

Reply:
But let's say the local storage is cleared, because the user went on the site for the first time, resuming from where they were before, and they have the flow token, how does that token helps to re-populate the state in memory and in local storage?

Unless I'm missing something, this seems to only work if they do the sign up all in one go and had the state stored in local storage already?

Reply:
If they're on a different device how would they have the token?

Reply:
I believe they get it in the email in the old API's sign up flow, and by doing so, it also confirms their email since it's implied that if they have access to the token, they have access to the email

Reply:
Ah I see. With the new API the email is only sent after the signup flow is completed

Reply:
It might be worth chatting with the backend team about this on another issue, as I think the whole point of having this flow token thing is so that user can resume sign up across devices or if they cleared their local storage/cookies. And I think that's meant to be a key feature of the sign up flow.
Otherwise, we may as well just store the WIP user ID we get from `POST /users` or something and keep patching the user object... with the caveat that if they clear their local storage/cookies, that user will effectively be "lost".

Reply:
resolving, see conversation here -> https://github.com/Couchers-org/backend-v2/issues/60